js/scripts.js

```diff
@@ -1526,6 +1526,24 @@ function plugin_formcreator_changeGlpiObjectItemType() {
    });
 }
 
+
+function plugin_formcreator_changePluginFieldBlock() {
```

Comment:
Please move this function in plugin_formcreator

Reply:
cod eremoved after refactoring with twig